### PR TITLE
BUILD: buildifier formatting

### PR DESCRIPTION
**Pull request by tburgin**
What do you think about adding buildifier to Santa's WORKSPACE?

https://github.com/bazelbuild/buildtools/tree/master/buildifier

### CODE REVIEW

**Comment by russellhancox** [comment]
> What do you think about adding buildifier to Santa's WORKSPACE?

I dunno, seems a bit overkill given how much needs to be added and maintained. Let's come back to that if we keep having to submit buildifier fixes 😄

File: BUILD [hunk 1]
```diff
@@ -177,11 +176,11 @@ objc_library(
     sdk_dylibs = ["libz"],
     sdk_frameworks = ["IOKit"],
     deps = [
-        "@FMDB//:fmdb",
-        "@MOLAuthenticatingURLSession//:MOLAuthenticatingURLSession",
-        "@MOLCodesignChecker//:MOLCodesignChecker",
-        "@MOLFCMClient//:MOLFCMClient",
-        "@MOLXPCConnection//:MOLXPCConnection",
+        "@FMDB",
```
Comment (russellhancox) [inline BUILD]: This fails - the rule inside the FMDB package is fmdb not FMDB.

Comment (tburgin) [reply BUILD]: Did you see the change to the WORKSPACE? It works for me...and Travis.

Comment (russellhancox) [reply BUILD]: Oh the WORKSPACE change, nevermind.

**Comment by tburgin** [comment]
>Let's come back to that if we keep having to submit buildifier fixes.
Agreed

**Pull request merged**